Makefile

```diff
@@ -231,38 +234,3 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
-
-
-# CI tools repo for running tests
-CI_TOOLS_REPO := https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci
-CI_TOOLS_REPO_DIR = $(shell pwd)/CI_TOOLS_REPO
-.PHONY: get-ci-tools
-get-ci-tools:
-	if [ -d  "$(CI_TOOLS_REPO_DIR)" ]; then \
-		echo "Ci tools exists"; \
-		pushd "$(CI_TOOLS_REPO_DIR)"; \
-		git pull --rebase; \
-		popd; \
-	else \
-		git clone $(CI_TOOLS_REPO) "$(CI_TOOLS_REPO_DIR)"; \
-	fi
-
-# Run go fmt against code
```

Comment:
we proablly want to keep this since we added them for ci i guess
ill revert this part